pkg/apis/serving/register.go

```diff
@@ -47,6 +47,10 @@ const (
 	// triggered their creation.
 	RouteLabelKey = GroupName + "/route"
 
+	// RoutesAnnotationKey is an annotation attached to a Revision to indicate that it is
+	// referenced by one or many routes. The value is a comma separated list of Route names.
+	RoutesAnnotationKey = GroupName + "/routes"
+
```

Comment:
Do we need both? Can one supercede the other?

Reply:
We need to allow both and pay attention to either because during the switchover to the new labeler, the old items will be converted.

https://github.com/knative/serving/blob/4a4711d39a7f97c1d1cf3daf8c971a127808033a/pkg/reconciler/route/resources/ingress.go#L69
https://github.com/knative/serving/blob/f1582404be275d6eaaf89ccd908fb44aef9e48b5/pkg/reconciler/route/resources/certificate.go#L70

There are also cases where route is 1:1 with the resource and we will need to be careful when considering those too. So I think they will both exist for a time.